### PR TITLE
Add documentation about RESTEasy Reactive Jackson features

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

File: docs/src/main/asciidoc/resteasy-reactive.adoc [hunk 1]
```diff
@@ -698,6 +698,100 @@ In both cases, importing those modules will allow HTTP message bodies to be read
...
+    return testUser();
+}
+----
+
+When the result the `userPublic` method is serialized, the `id` field will not be contained in the response as the `Public` view does not include it.
+The result of `userPrivate` however will include the `id` as expected when serialized.
+
+===== Completely customized per method serialization
+
+There are times when you need to completely customize the serialization of a POJO on a per JAX-RS method basis. For such use cases, the `@io.quarkus.resteasy.reactive.jackson.CustomSerialization`
```
Comment (gsmet) [inline docs/src/main/asciidoc/resteasy-reactive.adoc:763]:
```suggestion
There are times when you need to completely customize the serialization of a POJO on a per JAX-RS method basis. For such use cases, the `@io.quarkus.resteasy.reactive.jackson.CustomSerialization` annotation
```

File: docs/src/main/asciidoc/resteasy-reactive.adoc [hunk 2]
```diff
@@ -698,6 +698,100 @@ In both cases, importing those modules will allow HTTP message bodies to be read
...
+----
+@CustomSerialization(UnquotedFields.class)
+@GET
+@Path("/invalid-use-of-custom-serializer")
+public User invalidUseOfCustomSerializer() {
+    return testUser();
+}
+----
+
+where `UnquotedFieldsPersonBiFunction` is defined as so:
```
Comment (gsmet) [inline docs/src/main/asciidoc/resteasy-reactive.adoc:778]:
```suggestion
where `UnquotedFields` is a `BiFunction` defined as so:
```

File: docs/src/main/asciidoc/resteasy-reactive.adoc [hunk 3]
```diff
@@ -698,6 +698,100 @@ In both cases, importing those modules will allow HTTP message bodies to be read
...
+----
+public class Views {
+
+    public static class Public {
+    }
+
+    public static class Private extends Public {
+    }
+}
+----
```
Comment (gsmet) [inline docs/src/main/asciidoc/resteasy-reactive.adoc:736]: Shouldn't you declare the views before using them?

**Review by gsmet — CHANGES_REQUESTED** [review]
Added some small comments.

File: docs/src/main/asciidoc/resteasy-reactive.adoc (hunk 3 above)
Comment (geoand) [reply docs/src/main/asciidoc/resteasy-reactive.adoc:736]: Good point, I'll update

Comment (geoand) [reply docs/src/main/asciidoc/resteasy-reactive.adoc:736]: Fixed

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
@geoand do you want it merged and backported even without Stéphane's review or should it wait for the next round?

**Comment by geoand** [comment]
Sure yeah, it's very minor so it doesn't need a review from Stephane

**Review by FroMage — COMMENTED** [review]
LGTM.

**Pull request merged**